src/app/cli/src/explore/web_ui_server.rs

```diff
@@ -59,7 +59,7 @@ struct WebUILoginInstructions {
 struct WebUIFeatureFlags {
     enable_logout: bool,
     enable_scheduling: bool,
-    enable_dataset_env_vars_managment: bool,
+    enable_dataset_env_vars_management: bool,
```

Comment:
Don't just change it here, make a PR in Web UI repo.

Reply:
Extracted into a dedicated issue: https://github.com/kamu-data/kamu-cli/issues/841

Reply:
The change was reverted